tests/src/test/java/alluxio/client/fs/LoadMetadataIntegrationTest.java

```diff
@@ -101,63 +102,95 @@ public void after() throws Exception {
     ServerConfiguration.reset();
   }
 
+  @Test
+  public void syncOverrideLoadMetadata() throws Exception {
+    GetStatusPOptions options =
+        GetStatusPOptions.newBuilder().setLoadMetadataType(LoadMetadataPType.NEVER)
+            .setCommonOptions(FileSystemMasterCommonPOptions.newBuilder().setSyncIntervalMs(0)
+        ).build();
+    // The first time, it needs status of /dir1, /dir1/dirA to create the dirs in Alluxio
+    // and the file itself
+    checkGetStatus("/mnt/dir1/dirA/file", options, true, 3);
+    // The second time, it only needs to sync the file, so 1 access
+    checkGetStatus("/mnt/dir1/dirA/file", options, true, 1);
+  }
+
+  @Test
+  public void absentCache() throws Exception {
+    GetStatusPOptions options =
+        GetStatusPOptions.newBuilder().setLoadMetadataType(LoadMetadataPType.ONCE)
+            .setCommonOptions(FileSystemMasterCommonPOptions.newBuilder().setSyncIntervalMs(-1)
+        ).build();
+    checkGetStatus("/mnt/dir1/dirA/dirDNE/", options, false, 1);
+    checkGetStatus("/mnt/dir1/dirA/fileDNE1", options, false, 1);
+    checkGetStatus("/mnt/dir1/dirA/dirDNE/", options, false, 0);
+    checkGetStatus("/mnt/dir1/dirA/fileDNE1", options, false, 0);
+    checkGetStatus("/mnt/dir1/dirA/dirDNE/fileDNE", options, false, 0);
```

Comment:
can you add

`checkGetStatus("/mnt/dir1/dirA/dirDNE/dirB/fileDNE", options, false, 0);`